mise.toml

```diff
@@ -23,3 +30,33 @@ run = """
   rm -f release/cipherstash-encrypt-uninstall.sql
   rm -f release/cipherstash-encrypt.sql
 """
+
+[tasks."test:legacy"]
+description = "Run legacy SQL tests (inline test files)"
+sources = ["src/**/*_test.sql", "tests/*.sql"]
+run = "{{config_root}}/tasks/test-legacy.sh"
```

Comment:
This is a confusing way of doing this. 

We already have `tasks/test-legacy.sh`, but we're shadowing that with this TOML task definition. 

`tasks/test-legacy.sh` should be moved to `tasks/test/legacy.sh`, and this TOML task definition can be removed.